stdlib-full-bootstrapped/README.md

```diff
@@ -0,0 +1,13 @@
+## stdlib-full-bootstrapped
+
+This project compiles the Scala standard library using the Scala 3 compiler.
+It uses the sources Scala (2/3) standard library sources from `stdlib-bootstrapped/src`
+and the Scala 3 library form `library/src`. It fetches any missing Scala (2/3) sources
+from `org.scala-lang" % "scala-library" % stdlibVersion(Bootstrapped) % "sourcedeps`.
+This implies that files in `stdlib-bootstrapped/src` override files override the Scala (2/3)
+sources. Currently we can not override files from `library/src`.
+
+To clone or list non-overridden sources execute `stdlib-bootstrapped/run`.
```

Comment:
Should that be `stdlib-full-bootstrapped/run`?

Reply:
Right now, no. But it might be something that I might change. I am still working out the ideal structure of the projects and how to interact with them.